.github/workflows/release-publish.yaml

```diff
@@ -64,6 +64,8 @@ jobs:
       - name: Release
         uses: softprops/action-gh-release@v2
         with:
+          # Generate release notes on the new GH release
+          generate_release_notes: true
```

Comment:
This looks like it comes from our changes to `main`, right? If so, could you separately merge `main` to this feature branch so the PR is cleaner?

Reply:
@lengau I put the changes from main in seperate PR #582. Please merge that one first.